docs/src/examples/rotations.md

```diff
@@ -0,0 +1,141 @@
+# Rotations with quaternions
+
+One of the most useful application of quaternions is representation of 3D-rotations.
+See also [Rotations.jl documentation](https://juliageometry.github.io/Rotations.jl/stable/3d_quaternion/)
+
+```@example rotation
+using Quaternions
+using LinearAlgebra
+```
+
+## Basics
+A 3D rotation can be represented by a [unit quaternion (versor)](https://en.wikipedia.org/wiki/Versor).
+For example, 90° rotation around y-axis is ``q = 1/\sqrt{2} + 0i + j/\sqrt{2} + 0k``.
+Rotations with quaternions have the following properties:
+
+* An additive inverse of a unit quaternion represents the same rotation.
+* Unit quaternion is more efficient to represent rotation than rotation matrix.
+    * High computation accuracy
+    * High computation performance, both in time and memory
+* A conjugate of a unit quaternion represents the inverse rotation.
+    * The quaternion has unit length, so conjugate and multiplicative inverse is the same.
+* The set of unit quaternion ``\left\{w + ix + jy + kz \in \mathbb{H} \ | \ x, y, z \in \mathbb{R} \right\} \simeq S^3`` is isomorphic to ``SU(2)``.
+    * These groups are homomorphic to ``SO(3)``.
```

Comment:
We probably need to define SO(3). Can this and the below statement be merged?

Reply:
Done!